pages/episodes/[slug].vue

```diff
@@ -15,32 +15,32 @@ useHead({
   bodyAttrs: {
     class: 'has-head-color'
   }
-})
-const dataLoaded = ref(false)
-const episode = ref([])
+} )
+const dataLoaded = ref( false )
+const episode = ref( [] )
```

Comment:
only 1 episode will be in the array, so I think it's more descriptive this way

Reply:
why is there an array with only one episode? Is this something that the api does? That's a very odd pattern.

Reply:
I think a better way to handle this would be to unpack the episode in the `.then` block of the axios request. That way the name `episode` wouldn't be confusingly associated with an array. It would also allow us to take advantage of the existing exception handling, because if the api was acting unexpectedly and returned an empty array, the js equivalent of an index out of bounds exception would be thrown, triggering the `.catch` block.

Reply:
spoke to Bono and he also prefers "episode", so we'll leave it as is :)